java-security/src/main/java/com/sap/cloud/security/config/k8s/K8sEnvironment.java

```diff
@@ -0,0 +1,178 @@
+package com.sap.cloud.security.config.k8s;
+
+import com.sap.cloud.security.config.Environment;
+import com.sap.cloud.security.config.OAuth2ServiceConfiguration;
+import com.sap.cloud.security.config.OAuth2ServiceConfigurationBuilder;
+import com.sap.cloud.security.config.Service;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.*;
+import java.util.function.UnaryOperator;
+import java.util.stream.Collectors;
+
+import static com.sap.cloud.security.config.Service.IAS;
+import static com.sap.cloud.security.config.Service.XSUAA;
+
+/**
+ * Loads the OAuth configuration ({@link OAuth2ServiceConfiguration}) of a
+ * supported identity {@link Service} in the Kubernetes Environment by
+ * accessing defaults service secrets paths "/etc/secrets/sapcp/xsuaa" for Xsuaa service or "/etc/secrets/sapcp/ias" for .
+ * IAS service
+ */
+public class K8sEnvironment implements Environment {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(K8sEnvironment.class);
+
+    private static K8sEnvironment instance;
+    private static final String DEFAULT_XSUAA_PATH = "/etc/secrets/sapcp/xsuaa";
```

Comment:
should we name it sapcp or sapbtp?

Reply:
@nenaraab i guess this docu is then outdated https://wiki.wdf.sap.corp/wiki/display/PFS/Guidelines+for+SAP+CP+Service+Libraries 
to me it also makes sense to rename it to sapbtp to be inline with the new name

Reply:
@nenaraab I would propose to rename it in separate PR as it affects all later created PRs

Reply:
sure 👍